net/dns/publicdns/publicdns.go

```diff
@@ -183,12 +183,10 @@ func populate() {
 
 	// Mullvad
 	addDoH("194.242.2.2", "https://doh.mullvad.net/dns-query")
```

Comment:
The linked post suggests that they are moving from `doh.mullvad.net` to `dns.mullvad.net` and that `doh` is deprecated and will be retired. Might as well update the URL while you're in here.
Same for the adblock ones below.

Reply:
Good call, thank you